docs/layers.md

```diff
@@ -434,21 +842,21 @@ Railway values in this layer include: `rail`, `tram`, `light_rail`, `narrow_gaug
 
 ![image](images/mapzen-vector-tile-docs-roads-airport.png)
 
-**Airport** aeroways with `kind` values of `runway` show up at zoom 9, with `taxiway` at zoom 11+.
+**Airport** aeroways with `subkind` values of `runway` show up at zoom 9, with `taxiway` at zoom 11+.
```

Comment:
We don't have anything with `subkind` yet, do we? I remember we discussed it, but I can't find a ticket with any details about exactly what change we want to make (i.e: test cases).